server/routes/nl/html.py

```diff
@@ -52,14 +53,17 @@ def screenshot():
 
 @bp.route('/data')
 def data_page():
+  logging.info('NL Data Page: Enter')
```

Comment:
did you mean to keep these Enter and Exit logs? or were they for testing?

also in api.py